package.json

```diff
@@ -14,7 +14,8 @@
 			"import": "./dist/color.js",
 			"require": "./dist/color.cjs"
 		},
-		"./fn": "./src/index-fn.js"
+		"./fn": "./src/index-fn.js",
+		"./fn/spaces": "./src/spaces/index-fn.js"
```

Comment:
I'd expect `index-fn` to also import `spaces/index-fn`, so this shouldn't be needed. But not a strong opinion.

Reply:
Done in 05bdd32631c2100ea7d1362584f340cb4e7a4b12.

I also added a `sideEffects` declaration to `package.json` to ensure that tree-shaking works correctly.